Makefile

```diff
@@ -4,17 +4,23 @@ build:
 	docker build -t remind101/stacker .
 
 lint:
-	flake8 --exclude stacker/tests/ stacker
-	flake8 --ignore N802 stacker/tests # ignore setUp naming
+	flake8 --ignore E402 --exclude stacker/tests/ stacker
```

Comment:
Because future monkey patched some of the builtins for compatibility reasons. The code would look like this(except from `stacker/config/__init__.py`):

```
from __future__ import print_function
from __future__ import division
from __future__ import absolute_import
from future import standard_library
standard_library.install_aliases()
from builtins import str
import copy
import sys
```

flake8 would consider the last three imports should happen before `standard_library.install_aliases()`, wherein this case, since we are monkey patching, we cannot do what flake8 suggests.

We could alternatively disable E402 check in the code by writing `# noqa: E402` after the `install_aliases` call. Currently, I'm choosing the lazy way of disabling that check globally. I can add all these `noqa` lines if you want. :)

Reply:
Oh, sorry, I was confused - mixed it up with N802 above.  This makes sense, and I think I'm ok just keeping it like you have it now.  Thanks for the explanation.